src/main/kotlin/com/avail/descriptor/bundles/MessageBundleTreeDescriptor.kt

```diff
@@ -258,7 +279,7 @@ class MessageBundleTreeDescriptor private constructor(
 		 * treated specially, as only one keyword can be next in the source
 		 * stream (so there's no value in checking whether it's an X, whether
 		 * it's a Y, whether it's a Z, etc. Instead, the [LAZY_INCOMPLETE] and
-		 * [LAZY_INCOMPLETE_CASE_INSENSITIVE]&#32;[map][A_Map] takes care of
+		 * [LAZY_INCOMPLETE_CASE_INSENSITIVE] [map][A_Map] takes care of
```

Comment:
This is correct, but what was there before is interesting!

Reply:
Yeah, somebody regex'd wrong. 🙂